docs/installation.md

```diff
@@ -92,14 +89,17 @@ If you don't want to use this command and set the key yourself, this string shou
 **If the application key is not set, your user sessions and other encrypted data will not be secure!**
 
 #### Additional Configuration
-Lychee needs almost no other configuration out of the box. You are free to get started developing! However, you may wish to review the `config/app.php` file and its documentation. It contains some options that aren't included from `.env` that you may wish to change according to your installation.
 
 You may also want to configure a few additional components of Lychee in `.env`, such as:
 
 - Cache
 - Database
 - Session
 
+They are documented in the file if you copied the `.env.example` file.
```

Comment:
Why "if you copied the ... file"? That's not optional, is it? Maybe, it should rephrase more like "copy the file" followed by "make the necessary adjustments".

Reply:
As I learned just recently, the file is always copied by Composer, see https://github.com/LycheeOrg/LycheeOrg.github.io/pull/56#discussion_r889675863. Hence, the condition does make even less sense than it has made before.

In theory, you are right. If users create the file manually, they file won't contain any comments. But I think, this is nothing we should consider, because this somehow holds for everything. We should assume that the user follows the configuration guide. Otherwise we would also need to write conditions like "file xyz contains comments unless you have deleted them". If users deliberately do something else than written in the guide, then they should know what they are doing and won't need the comments anyway. This is in particularly true for the `.env` file. If users managed to write their own, working `.env` file from scratch without using the template, then they should also know that they file won't contain any comments unless they have written their own comments.

Reply:
Maybe
```suggestion
They are documented in the `.env.example` file, which may have been copied into your `.env`. They are also listed on the [configuration](configuration.html) page.
```
?

Reply:
@nagmat84's last comment arrived while I was reviewing. I don't mind assuming the comments are there, but I think it makes sense to link to the configurations table added in this PR.